pycqed/analysis/analysis_toolbox.py

```diff
@@ -1278,17 +1279,17 @@ def color_plot(x, y, z, fig, ax, cax=None,
     if plot_title is not None:
         ax.set_title(plot_title, y=1.05, fontsize=18)
     if transpose:
-        ax.set_xlabel(ylabel)
-        ax.set_ylabel(xlabel)
+        set_xlabel(ax, ylabel, unit=y_unit)
```

Comment:
Related to #226 , it is better to set the lables after setting the axis limits or all the way at the end of the plotting. This is only a workaround but does do the job we expect it to.